sliceguard/data.py

```diff
@@ -1,6 +1,9 @@
+from typing import List
+from pathlib import Path
 import pandas as pd
 import datasets
 from datasets import Image, ClassLabel, Value, Sequence
+from bing_image_downloader import downloader
```

Comment:
If going for a tutorial dependency group make this an optional dependency that is requested only when called.